core/services/validators.go

```diff
@@ -192,5 +192,11 @@ func ValidateServiceAgreement(sa models.ServiceAgreement, store *store.Store) er
 			time.Now().Add(config.MinimumServiceDuration())))
 	}
 
+	if a, _ := models.NewEIP55Address("0x0000000000000000000000000000000000000000"); sa.Encumbrance.Aggregator == a {
+		return fmt.Errorf("please set aggregator contract")
+	}
+	// Can't do the above test for aggregator function selectors, because
+	// probability of zero value is non-trivial.
```

Comment:
Can you please elaborate on this?  I don't quiet understand.

Reply:
I just removed this, in 12b35c3db5982080f31ff923c704ca1663d2076b. I'll rely on the JSON parser to guard against empty aggregator fields.